editor/run/editor_run_native.cpp

```diff
@@ -99,9 +93,7 @@ void EditorRunNative::_confirm_run_native() {
 }
 
 Error EditorRunNative::start_run_native(int p_id) {
-	if (p_id < 0) {
-		return OK;
-	}
+	ERR_FAIL_COND_V(p_id < 0, FAILED);
 
 	int platform = p_id / 10000;
 	int idx = p_id % 10000;
```

Comment:
Sorry, just noticed now, but you defined `decode_platform_from_id`/`decode_device_from_id`, but didn't end up using them:
```suggestion
	int platform = decode_platform_from_id(p_id);
	int idx = decode_device_from_id(p_id);
```